android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/ReceiptCapturePlugin.kt

```diff
@@ -113,6 +113,19 @@ class ReceiptCapturePlugin : Plugin() {
     @ActivityCallback
     private fun onScanResult(call: PluginCall, result: ActivityResult) = receiptCapture.physical.onResult(call, result)
 
+//    TODO CALLBACK FOR GMAIL LOGIN
+//    @ActivityCallback
+//    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+//        super.onActivityResult(requestCode, resultCode, data)
+//        val gmailClient = GmailClient(this, googleId)
+//        gmailClient.onAccountAuthorizationActivityResult(requestCode, resultCode, data)
```

Comment:
This is probably where microblink register the callback from gmail. Move this line to the login function and it might work.